FunctionalPreprocessing/FunctionalPreprocessing.XNAT_MARK_RUNNING_STATUS

```diff
@@ -50,11 +50,15 @@ get_options()
 	local arguments=($@)
```

Comment:
I don't know how to make this comment show up with the lines that it should show up with.

The XNAT_PBS_JOBS_RUNNING_STATUS environment variable is no longer used by this script. So the code that checks to see if it is set near the beginning of this script should be removed.  In it's place is where I would put the check for other environment variables that now are used by this script: `XNAT_PBS_JOBS_BUILD_DIR` and `XNAT_PBS_JOBS_ARCHIVE_ROOT`.

Reply:
added